### ISSUE TITLE
Taskbar themes stop working in newer Windows 11 builds

### ISSUE THREAD

**Issue by truong1112** [rocksdanister/lively#1489]
I am a windows 11 user . For the past few months after an update i am not able to change the theme of my taskbar to clear . I have tried to uninstall and reinstall but nothing changed . Pls help .

**Comment by rocksdanister** [comment]
Yes, Windows changed the taskbar in newer Windows 11 builds so for the time being taskbar themes will be disabled.
